### PR TITLE
ACTIN-431 Copy bucket located pipeline outputs to archival buckets

### PR DESCRIPTION
**Pull request by pauldwolfe**
Adds locate functions to find all archived files in buckets rather than on VM filesystems.

@kduyvesteyn @spdeleeuw I think there is some redundancy between gs://actin-prod-patient-data, gs://actin-prod-evaluation, and gs://actin-prod-analysis-output. Can we consolidate to one bucket?

If the answer is yes, will probably abandon this PR and automate the archival on the GCP side.

### CODE REVIEW

**Comment by spdeleeuw** [comment]
Not sure if this is still relevant since you already had some discussion on the subject, but there seems to be some redundancy between those buckets indeed. “Actin-prod-evaluation” seems to contain all data for each patient, except the WGS-output. If there's no specific reason to keep these buckets separate (@kduyvesteyn ?), we could consolidate them I think. I'm not sure what to do with this info, but I noticed that the buckets contain the same info for different patients. For example: “Actin-prod-actin-analysis-output -> algo” and “actin-prod-patient-data” -> treatment_match” do not contain the exact same patients.

File: functions/locate_files [hunk 1]
```diff
@@ -426,47 +430,84 @@ locate_actin_prod_data_bucket() {
...
+locate_actin_analysis_bucket() {
+  echo "gs://actin-prod-actin-analysis-output"
 }
 
-locate_actin_curated_clinical_directory() {
-    echo "/data/actin/clinical/curated"
+################################# ACTIN Data Bucket Based ###################################
+
+locate_actin_blob(){
+  local gsutilUri=$1 && shift
```
Comment (kduyvesteyn) [inline functions/locate_files:440]: We use underscores to separate words in bash, should be `gsutil_uri`.

File: functions/locate_files [hunk 2]
```diff
@@ -426,47 +430,84 @@ locate_actin_prod_data_bucket() {
...
-    echo "/data/actin/clinical/input_feed/latest"
+locate_actin_analysis_bucket() {
+  echo "gs://actin-prod-actin-analysis-output"
 }
 
-locate_actin_curated_clinical_directory() {
-    echo "/data/actin/clinical/curated"
+################################# ACTIN Data Bucket Based ###################################
+
+locate_actin_blob(){
```
Comment (kduyvesteyn) [inline functions/locate_files:439]: Space missing after parentheses

**Review by kduyvesteyn — COMMENTED** [review]
I think it is all ok, though a fair bit of formatting issues (have mentioned a few). 

Also - assume things have been tested properly (that's usually the biggest challenge for script changes!)

**Pull request merged**